Problem2.py

```diff
@@ -0,0 +1,14 @@
+t = int(input())
+for i in range(1, t+1) :
+    n = int(input())
+    case = list(map(int,input().split()))
+    all_cost = 0
+    avg = 0
+    cnt = 0
+    for x in case :
+        all_cost +=x
+    avg = all_cost //len(case)
```

Comment:
깔끔하네요! 👍 
```suggestion
    avg = all_cost / len(case)
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다 😄